tony-core/src/main/java/com/linkedin/tony/TonyClient.java

```diff
@@ -484,12 +485,23 @@ public void processTonyConfResources(Configuration tonyConf, FileSystem fs) thro
             }
           } else {
```

Comment:
Can you add a comment here `// file is directory`?

Reply:
updated

Reply:
updated, thanks